### PR TITLE
[10.0][FIX] Fixes to Website Portal Medical and ~Patient

### PR DESCRIPTION
**Pull request by woodbrettm**
* website_portal_medical:
    * Fix js assets link
* website_portal_medical_patient:
    * Fix patient compute website url method
    * Add respective test case
    * Fix occasional underline of patient name when clicking image

### CODE REVIEW

**Comment by lasley** [comment]
Rebased onto release/10.0, which I think should resolve the build error.

File: website_portal_medical_patient/tests/test_medical_patient.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        super(TestMedicalPatient, self).setUp()
+        self.patient_1 = self.env.ref(
+            'website_portal_medical_patient.medical_patient_1'
+        )
+
+    def test_compute_website_url(self):
+        """ Test website_url returns correct id """
+        self.assertEquals(
+            self.patient_1.website_url,
+            '/medical/patients/%s' % (self.patient_1.id),
```
Comment (tedsalmon) [inline website_portal_medical_patient/tests/test_medical_patient.py]: No need to wrap a single parameter in parenthesis.

**Review by tedsalmon — APPROVED** [review]
LGTM! Tested in runbot and it works fine.

**Comment by woodbrettm** [comment]
Thanks @tedsalmon, updated.

**Review by lasley — APPROVED** [review]

**Pull request merged**